torchgeo/datamodules/so2sat.py

```diff
@@ -25,14 +25,6 @@ class So2SatDataModule(pl.LightningDataModule):
 
     band_means = torch.tensor(
         [
-            -3.591224256609313e-05,
```

Comment:
Our previous implementation here didn't really make sense. We were loading all band data, doing channel-wise standardization over all bands, then throwing away the S1 bands so I removed loading of the S1 bands.